southbound/impl/src/main/java/org/opendaylight/ansible/southbound/AnsibleCommandServiceImpl.java

```diff
@@ -215,4 +239,35 @@ public ManagedProcess getProcess(Uuid uuid) {
         }
         return null;
     }
+
+    public Status getAnsibleResult(Uuid uuid) {
+        Optional<Command> optCmdResult;
+        CommandKey cmdKey = new CommandKey(uuid);
+        InstanceIdentifier<Command> cmdPath = InstanceIdentifier.create(Commands.class).child(Command.class, cmdKey);
+        try {
+            optCmdResult = SingleTransactionDataBroker.syncReadOptional(dataBroker, LogicalDatastoreType.OPERATIONAL,
```

Comment:
in future we should use the runner.

Reply:
im not sure with runner how to get the result outside of the transaction context. Can you provide an example?

Reply:
there is an apply API - but that is not ported in yet so it isn't available. But we could do the same. No biggie right now. The runners are just better perf which we don't need right now.